sphinx_scality/static/css/current.css_t

```diff
@@ -247,21 +298,148 @@ div.footer div.leftwrapper {
     display: flex;
     flex-direction: column;
     align-items: flex-start;
+    margin: 1% 8% 0 7%;
 }
 
 div.footer div.links {
-    font-size: 1.8em;
-    margin-top: -10px;
+    font-size: 1.1em;
+    margin-top: 15px;
 }
 
 div.footer div.rightwrapper {
     display: flex;
     flex-grow: 1;
-    padding-right: 15px;
     justify-content: flex-end;
     align-items: center;
+    padding-right: 20%;
+    padding-top: 2%;
 }
 
 div.footer div.rightwrapper img.social {
-    margin: auto 15px;
+    margin: auto -1px;
+    width: 29%;
+}
+
+img.baselogo.white {
+    width: 117px;
+}
+
+/*Search results*/
+.term {
+    color: {{ theme_termcolor }};
+    font-weight: bold;
+}
+
+/*Search box*/
+input[type=text] {
```

Comment:
Oops, we need more specificity on this selector. So, two changes to do:

1. In `sphinx_scality/searchbox.html`:

```diff
 {%- if builder != 'singlehtml' %}
 <div role="search">
-  <form id="rtd-search-form" class="wy-form" action="{{ pathto('search') }}" method="get">
+  <form id="search-form" class="wy-form" action="{{ pathto('search') }}" method="get">
     <input type="text" name="q" placeholder="{{ _('Search docs') }}" />
     <input type="hidden" name="check_keywords" value="yes" />
     <input type="hidden" name="area" value="default" />
   </form>
 </div>
 {%- endif %}
```

2. Given this slight adjustment (not critical, but `rtd-` prefix is for ReadTheDocs :wink:), you can use a more specific selector, in this file:

```suggestion
form#search-form > input[name=q] {
```